lib/appium_lib/driver.rb

```diff
@@ -319,9 +319,9 @@ def initialize opts={}
       @appium_debug = appium_lib_opts.fetch :debug, !!defined?(Pry)
 
       if @appium_debug
-        ap opts unless opts.empty?
-        puts "Debug is: #{@appium_debug}"
-        puts "Device is: #{@appium_device}"
+        Appium::Logger.ap opts unless opts.empty?
```

Comment:
This should probably remove the `@appium_debug` construct and then invoke debug on the logger.